ansible/dashboard-web.yml

```diff
@@ -25,6 +25,14 @@
   - { role: software/dashboard/dashboard-deploy, tags: [deploy, provision] }
 
 
+- name: Dashboard DB Migration
+  hosts: dashboard-web
+  serial: 1
+  become: no
+  roles:
+  - { role: software/dashboard/dashboard-db-migrations, tags: [provision, deploy, migrate] }
```

Comment:
Will this break? I know we do some permissions changes in the clean-up step, so I would expect that migrations could fail to execute application code until the permissions are fixed... but maybe we fixed the permission issue on the app code with the `umask` change :man_shrugging:

Reply:
... also, can we capture this in an issue: Dashboard should run the migrations on startup instead of it being part of deploy. I _think_ the startup script that @pburkholder added already does this in cloud.gov, but we want it to work the same here so that new migrations work consistently between environments.

Reply:
I don't _think_ it will break. How can I tell other than running it in CI as I've been testing it?

And yes, I'll capture the "dashboard always runs its own migrations" in a separate issue.

Reply:
If it were to break, it would probably break at the shell level running the command, with permission denied. If you've seen it run successfully, I think it's fine.

Reply:
> Dashboard should run the migrations on startup instead of it being part of deploy.

I've just made a change so [the dashboard does migrations on its own](https://github.com/GSA/project-open-data-dashboard/commit/2e376fbf3f3e3494da774335e3be0114fe02ee52), and [removed the migration task referenced here](https://github.com/GSA/datagov-deploy/commit/b9240af080fe24f7686a98f988d0487de34fa1aa). So no issue necessary!